webapi/tct-canvas-html5-tests/canvas-py/spec.json

```diff
@@ -1,6 +1,7 @@
 {
   "canvas":{
     "spec_url":"http://www.w3.org/TR/2012/WD-2dcontext-20120329/",
+    "spec_maturity": "CR",
```

Comment:
This version is WD; the latest is CR.

Can you please check the spec link carefully?? Please double check them all.